Tests/TestData/SelfVerifyingTests/CSToVB/StringEqualityTests.cs

```diff
@@ -10,10 +10,17 @@ public class StringEqualityTests
     private char[] emptyCharArray = new char[0];
 
     [Fact]
-    public void NullEmptyStringEqualsOperator() => Assert.False(nullString == emptyString);
+    public void NonInternStringsEqualsOperator() {
+        string a = "10";
+        string b = 10.ToString(); //strings created in runtime are not interned
+        Assert.True(a == b);
+    }
```

Comment:
Yep, I like this test, it shows that op_Equality is overridden and distinguishes from reference equality (which neither VB or C# does, but someone might guess from "==")